openpype/tools/sceneinventory/view.py

```diff
@@ -784,7 +784,7 @@ def _show_version_error_dialog(self, version, items):
         else:
             version_str = version
 
-        dialog = QtWidgets.QMessageBox()
+        dialog = QtWidgets.QMessageBox(self)
```

Comment:
This looks like a quickfix which could already be merged if would be in different PR.

Reply:
PR : https://github.com/ynput/OpenPype/pull/4941